iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/v1/models/DeploymentMetricsApiModel.java

```diff
@@ -46,7 +46,7 @@ public DeploymentMetricsApiModel(DeploymentMetrics metricsServiceModel) {
         if (metricsServiceModel == null) return;
 
         this.customMetrics = metricsServiceModel.getCustomMetrics();
-        this.systemMetrics = MapUtils.isEmpty(metricsServiceModel.getSystemMetrics()) ?
+        this.systemMetrics = !(MapUtils.isEmpty(metricsServiceModel.getSystemMetrics())) ?
```

Comment:
nit: combine line 30 here.

Reply:
same argument as .Net, we want to have default values before setting the metrics from the model.